analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -1082,9 +1082,9 @@ private void ConditionalAccess_NullCoalescing()
             string someString = null;
 
             if (!someString?.Contains("a") ?? true)
-                Console.WriteLine("It's null or doesn't contain 'a'");
+                someString.ToString();  // Noncompliant, it's null or doesn't contain 'a'
             else
-                Console.WriteLine(someString.Length); // Noncompliant FP, this path is unreachable
+                someString.ToString();  // Compliant, this path is unreachable
```

Comment:
```suggestion
                someString.ToString();  // Compliant, someString is not null here
```

Reply:
No, it is really unreachable, because we have `string someString = null` at the beginning.